experimental/dds/tree2/src/test/tree/anchorSet.spec.ts

```diff
@@ -321,8 +313,14 @@ describe("AnchorSet", () => {
 		anchors.applyDelta(makeDelta(deleteMark, makePath([rootFieldKeySymbol, 0], [fieldFoo, 5])));
 		log.expect([["visitSubtreeChange.onDelete-foo-5-1", 1]]);
 		log.clear();
-		anchors.applyDelta(new Map([[rootFieldKeySymbol, [0, valueMark]]]));
-		log.expect([["visitSubtreeChange.onSetValue-xyz", 1]]);
+		anchors.applyDelta(makeDelta(deleteMark, makePath([rootFieldKeySymbol, 0], [fieldFoo, 0])));
+		anchors.applyDelta(
+			makeDelta(insertXYZMark, makePath([rootFieldKeySymbol, 0], [fieldFoo, 0])),
+		);
+		log.expect([
+			["visitSubtreeChange.onDelete-foo-0-1", 1],
+			["visitSubtreeChange.onInsert-foo-0", 1],
+		]);
```

Comment:
This is already tested with the `log.expect([["visitSubtreeChange.onInsert-foo-4", 1]]);` and `log.expect([["visitSubtreeChange.onDelete-foo-5-1", 1]]);` so you can just remove it.